README.md

```diff
@@ -1,78 +1,103 @@
+This is the official BigCommerce API client to support our Stores API. You can find more information about becoming a BigCommerce developer here: [developer.bigcommerce.com](http://developer.bigcommerce.com).
 
 
 ## Installation
-Bigcommerce is available on Rubygems:
+BigCommerce is available on Rubygems:
 
 ```sh
-gem install bigcommerce --pre
+gem install bigcommerce
 ```
 
 You can also add it to your Gemfile.
 
 ```rb
-gem 'bigcommerce', '>= 1.0.0.beta'
+gem 'bigcommerce', '~> 1.0.0'
 ```
 
+## Requirements
+Must be running Ruby 2.0.0 or newer. Please refer to the `.travis.yml` to see which versions we officially support.
+
 ## Getting Started
 In order to make requests to our API, you must register as a developer and have your credentials ready.
 
 We currently have two different authentication schemes you can use depending on your use-case.
 
 #### Public Apps
-Public apps can be submitted to Bigcommerce App Store, allowing other businesses to install it in their Bigcommerce stores.
+Public apps can be submitted to BigCommerce App Store, allowing other merchants to install it in their BigCommerce stores.
 
-[More Information](https://developer.bigcommerce.com/api/using-oauth-intro)
+__[More Information](https://developer.bigcommerce.com/api/using-oauth-intro)__
 
-#### Private Apps
+#### Private Apps (Legacy Auth)
 To develop a custom integration for one store, your app needs to use Basic Authentication.
 
-[More Information](https://developer.bigcommerce.com/api/legacy/basic-auth)
+__[More Information](https://developer.bigcommerce.com/api/legacy/basic-auth)__
 
 ## Usage
-For full examples on using the API client, please see the [examples folder](examples).
+For full examples on using the API client, please see the [examples folder](examples) and refer to the [developer documentation](https://developer.bigcommerce.com/api).
 
 ## Configuration
-In order to authenticate the API client, you will need to configure the client like this:
+In order to authenticate the API client, you will need to configure the client like the following.
 
-#### Single Click (Public Apps):
+#### Public Apps:
 
 - ```client_id```: Obtained from the "My Apps" section on the [developer portal](http://developer.bigcommerce.com).
 - ```access_token```: Obtained after a token exchange in the auth callback.
 - ```store_hash```: Also obtained after the token exchange.
 
 ```rb
+# Not thread safe
 Bigcommerce.configure do |config|
-  config.store_hash = 'store_hash'
-  config.client_id = 'client_id'
-  config.access_token = 'access_token'
+  config.store_hash = ENV['BC_STORE_HASH']
+  config.client_id = ENV['BC_CLIENT_ID']
+  config.access_token = ENV['BC_ACCESS_TOKEN']
 end
+
+Bigcommerce::System.time
+
+# Thread safe
```

Comment:
This should be clearer. What you're saying is that the API access methods are threadsafe as long as the application maintains the thread safety of the connection. If you want to make connections to multiple stores, or make multiple connections to one store, you need to set these connections up yourself and keep them in thread safe storage.